test/unit/commands/general-specs.js

```diff
@@ -181,7 +181,7 @@ describe('General', () => {
         .should.be.true;
     });
     it('should throw an error if APK does not exist', async () => {
-      await driver.installApp('non/existent/app.apk').should.be.rejectedWith(/does not exist or is not accessible/);
+      await driver.installApp('non/existent/app.apk').should.be.rejectedWith(/Could not find app apk at/);
```

Comment:
This change should be removed. It is broken. See https://travis-ci.org/appium/appium-android-driver/jobs/327567864#L6469